### DISCUSSION TITLE
Is there a way to change where the Config directory is stored?

### DISCUSSION THREAD

**Discussion by stevos758** [supermerill/SuperSlicer#1772] [Q&A]
I would like to be able to change where my config files are stored so I can keep them in my dropbox and use across multiple PCs at home and work.

**Comment by supermerill** [comment]
yes, see https://github.com/supermerill/SuperSlicer/issues/1752

```
You can access the dir via "help -> show conf folder"
If you have a "configuration" dir next to the exe, it will be used instead
You can choose the config dir via the --datadir command option, look at the superslicer_local-settings.bat in the win release
```
not optimal, but it will be improved.

**Comment by tome9111991** [comment]
You have to add your dropbox as network drive.
With --datadir you can change the path to your dropbox drive.

**Reply by ids444** [reply]
Use a symbolic link

Create a symbolic link using mklink that points to the folder in dropbox

ex. mklink /d %AppData%/Roaming/Superslicer {dropbox}/folder

**Reply by supermerill** [reply]
or `mklink /d %programfiles%/SuperSlicer/configuration {dropbox}/folder`

**Comment by TYTheBeast** [comment]
This is an old and deceased discussion, but it is still relevant for any NixOS users that would like to have their profiles elsewhere that isn't the Nix store, since the Nix store isn't writeable from the user or even from root.
Please update the discussion if you've added a new permanent configuration for the config folder.
